web-app/src/app/screens/Feeds/utility.ts

```diff
@@ -19,9 +21,10 @@ export function getInitialSelectedFeedTypes(
   const gtfsRtSearch = searchParams.get('gtfs_rt');
 
   if (gtfsSearch === null && gtfsRtSearch === null) {
+    // TODO: Change when GBFS is implemented
     return {
-      gtfs: false,
-      gtfs_rt: false,
+      gtfs: true,
+      gtfs_rt: true,
```

Comment:
we can keep this `false` `false` otherwise the page will default to have them selected. Since it our query function above handles the false false condition, it will still filter gbfs